public/tier0/dbg.h

```diff
@@ -315,13 +315,10 @@ PLATFORM_INTERFACE void COM_TimestampedLog( char const *fmt, ... ) FMTFUNCTION(
 //-----------------------------------------------------------------------------
 // Macro to assist in asserting constant invariants during compilation
 
-#ifdef _DEBUG

```

Comment:
Was the removal of #ifdef _DEBUG intentional?

Reply:
Yes, there's no need to keep it debug only.